cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -444,6 +444,11 @@ LOOP:
 
 func (mh *MessageHandle) send(hi hubio.CloudHubIO, info *model.HubInfo, msg *beehiveModel.Message) {
 	err := mh.hubIoWrite(hi, info.NodeID, msg)
+	if err.Error() == "node disconnected" {
```

Comment:
Has any difference with L42? :)

Reply:
The only difference is "has been closed" and "will bee closed".
I would suggest to updated the patch to avoid code duplication.

Reply:
@kevin-wangzefeng  How to update the patch to avoid code duplication? :)